scripts/package

```diff
@@ -17,10 +17,9 @@ if [ ! -e ./bin/longhorn-manager ]; then
     ./scripts/build
 fi
 
-cp -a ./bin ./package/
-cp -a ./driver ./package/
 trap 'rm -rf ./package/bin' exit
-docker build -t ${IMAGE} ./package
+
```

Comment:
Need to explain why you can remove the above two copy steps.

Reply:
Since i've changed the build context to be the root directory of the repo while still using package/Dockerfile to build the image. 

`docker build -t ${IMAGE} -f package/Dockerfile .`

There is no need now to copy `bin` and `driver` directory to `package` directory since they are already exits in the repo root directory.